admin/src/js/controllers/images-branding.js

```diff
@@ -96,7 +96,7 @@ angular.module('controllers').controller('BrandingCtrl',
       
     };
 
-    $scope.submitTitle = function() {
+    $scope.submitTitle = () => {
       $scope.error = null;
       if (!$scope.doc.title) {
         $scope.error = $translate.instant('field is required', {
```

Comment:
Usually we name the file to match the name of the controller so they're easy to find. Why did you rename the file? Should we update the controller name to match or review the file name?